### ISSUE TITLE
Get key

### ISSUE THREAD

**Issue by Naddox** [GoldenKappa/notSABS#2]
The instructions on how to get a key are out of date. Samsung has change the entire key process. Users need to scroll down on the generate license page to find Legacy SDKs and click on KNOX Standard SDK.

![2018-05-03_12-07-51](https://user-images.githubusercontent.com/845312/39591866-af5da978-4eca-11e8-9cfa-eb37b26a4ed2.png)

**Comment by Sillikk** [comment]
It seems it is not possible to generate legacy licenses keys at all now. I don't see the option anywhere.

**Comment by GoldenKappa** [comment]
Try this: [https://seap.samsung.com/license-keys](https://seap.samsung.com/license-keys)

Remember to login as well.

**Comment by Naddox** [comment]
Yeah they have removed them and now there is a notice saying support for legacy SDKs will be removed July 2019.

![2018-06-30_8-49-29](https://user-images.githubusercontent.com/845312/42125788-adbb16fa-7c42-11e8-95c8-16204a6ca8fe.png)

**Comment by aletheus** [comment]
I didnt have a problem getting a standard Knox SDK license key as a developer. It worked fine

**Comment by kannan1423** [comment]
It's not possible at all. Instructions need to be updated please ads are annoying af

**Comment by han9over** [comment]
Samsung removed support for legacy sdk key generation. So you cannot generate a new key if existing one expired. I don't want to promote different app, but addhell3 added a beta version with support for knox sdk. Notsabs has to be updated for it.
